src/Extension/DebugExtension.php

```diff
@@ -54,7 +54,7 @@ function twig_var_dump(Environment $env, $context, array $vars = [])
         return;
     }
 
-    ob_start();
+    ob_start(function ($in) {return ''; });
```

Comment:
Missing space before return.

Reply:
I suppose there is no need for the argument btw.

Reply:
fixed